Dev/VideoChatClient/Input/AudioInput.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <Windows.h>
```

Comment:
Include windows first, then the built in libs

Reply:
Sure